test/MUXControlsTestApp/VisualTreeTestHelper.cs

```diff
@@ -0,0 +1,322 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Common;
+using System;
+using System.Threading.Tasks;
+using Windows.Storage;
+using System.IO;
+using MUXControls.TestAppUtils;
+using MUXControlsTestApp.Utilities;
+using System.Text;
+using System.Collections.Generic;
+using Windows.UI.Xaml.Markup;
+using DiffPlex.DiffBuilder;
+using DiffPlex.DiffBuilder.Model;
+using static MUXControls.TestAppUtils.VisualTreeDumper;
+
+#if USING_TAEF
+using WEX.TestExecution;
+using WEX.TestExecution.Markup;
+using WEX.Logging.Interop;
+#else
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting.Logging;
+#endif
+
+using PlatformConfiguration = Common.PlatformConfiguration;
+
+namespace Windows.UI.Xaml.Tests.MUXControls.ApiTests
+{
+    public enum Theme
+    {
+        Dark,
+        Light,
+        All,
+        None
+    }
+    class VisualTreeLog
```

Comment:
```suggestion
// This class is used for internal debug purpose only. We don't have a flag to control log level in UnitTesting.Logging.
// Like debug information, it's too big and we should not show to customer. It will provide more information if you make small change on LogDebugInfo like this
// public static void LogDebugInfo(string debugInfo) { Log.Comment(info); }
    class VisualTreeLog
```